docs/python-unit-tests.md

```diff
@@ -60,33 +60,29 @@ You can select specific environments using `-e`.
 Running `tox` by itself is the same as running:
 
 ```sh
-tox -e lint -e unittest-py36-dj111-wag113-slow
+tox -e lint -e unittest-current
```

Comment:
Could we make this environment plain "unittest" and assume it's current?

Reply:
Absolutely, It'd just involve creating a new `testenv` that uses the `unittest-current` configuration. [That's what `lint` is](https://github.com/cfpb/cfgov-refresh/pull/5581/files#diff-b91f3d5bd63fcd17221b267e851608e8R160-R167)

Reply:
Done!